perspective-component/README.md

```diff
@@ -143,6 +144,7 @@ Within the `web` directory is a _lerna workspace_, which is simply a javascript
   └── web                                 // parent directory for the web assets we build
       ├── README.md
       ├── build.gradle
```

Comment:
Might as well remove this `build.gradle` from the tree here, and also convert the other `build.gradle` mentions above to `build.gradle.kts`.